server/repositories/cmsApi.js

```diff
@@ -15,10 +15,29 @@ class CmsApi {
     this.jsonApiClient = jsonApiClient;
   }
 
+  async lookupContent(establishmentName, contentId) {
+    const {
+      jsonapi: { resourceName, individual },
+    } = await this.jsonApiClient.getRelative(
+      `/router/prison/${establishmentName}/translate-path?path=content/${contentId}`,
+    );
+
+    return {
+      type: resourceName,
+      location: individual,
+    };
+  }
+
   async get(query) {
-    const response = await this.jsonApiClient.get(query.path());
-    const items = dataFormatter.deserialize(response);
-    return items.map(item => query.transform(item));
+    const request = query.path
+      ? this.jsonApiClient.getRelative(query.path())
+      : this.jsonApiClient.getUrl(query.url());
```

Comment:
allowing queries to either specify absolute or relative urls.

 `/content/:id` items will be looked up via the router url above and that will provide information about the type of item and also it's absolute location.